metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi.py

```diff
@@ -741,7 +742,7 @@ def report_to_dashboard(
 
         # DashboardInfo mcp
         dashboard_info_cls = DashboardInfoClass(
-            description=report.description or "",
+            description=report.description or str(),
```

Comment:
We can say `description=report.description` if we handle it at the time of object creation

Reply:
Fixed!